src/test/kotlin/no/nav/syfo/slowtests/SystemTestBase.kt

```diff
@@ -60,12 +59,12 @@ open class SystemTestBase : KoinTest {
      * Hjelpefunksjon for å hente ut gyldig JWT-token og legge det til som Auth header på en request
      */
     suspend fun HttpRequestBuilder.loggedInAs(subject: String) {
-        val response = httpClient.get<HttpResponse> {
-            appUrl("/local/cookie-please?subject=$subject")
+        val response = httpClient.get {
+            appUrl("/local/token-please?subject=$subject")
```

Comment:
<img src="https://media1.giphy.com/media/4BXO1nZSI7QDS/giphy.gif"/>

Reply:
Det var ren copy paste fra fritak den er egentlig ikke i bruk. Er jo ikke noe frontend for denne appen. Så fjerner den helt jeg.